<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PointerPropertiesWidget</class>
 <widget class="QWidget" name="PointerPropertiesWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>418</width>
    <height>420</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edge Properties</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Name:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>_name</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="KLineEdit" name="_name">
     <property name="toolTip">
      <string>Edge name</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2" rowspan="3">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Color:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>_color</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="KColorCombo" name="_color">
     <property name="toolTip">
      <string>Edge color</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Value:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>_value</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="KLineEdit" name="_value">
     <property name="toolTip">
      <string>Edge value</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QLabel" name="label_4">
     <property name="toolTip">
      <string>Edge width</string>
     </property>
     <property name="text">
      <string>Width:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>_width</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="KDoubleNumInput" name="_width"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Style:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>_style</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="KComboBox" name="_style">
     <property name="toolTip">
      <string>Edge style</string>
     </property>
     <item>
      <property name="text">
       <string>solid</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>dash</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>dot</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>dash dot</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Show</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KPushButton" name="_showName">
       <property name="toolTip">
        <string>Show/hide edge name</string>
       </property>
       <property name="text">
        <string>Name</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KPushButton" name="_showValue">
       <property name="toolTip">
        <string>Show/add edge value</string>
       </property>
       <property name="text">
        <string>Value</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="0" colspan="5">
    <widget class="QFrame" name="_extraProperties">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
    </widget>
   </item>
   <item row="5" column="0" rowspan="2" colspan="2">
    <widget class="QTableView" name="_propertiesTable"/>
   </item>
   <item row="6" column="2" colspan="3">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Add Property</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0" colspan="2">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Name</string>
        </property>
        <property name="buddy">
         <cstring>_propertyName</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="KLineEdit" name="_propertyName"/>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Value</string>
        </property>
        <property name="buddy">
         <cstring>_propertyValue</cstring>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="KLineEdit" name="_propertyValue"/>
      </item>
      <item row="4" column="0">
       <widget class="QCheckBox" name="_isPropertyGlobal">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Global</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="5" column="1">
       <widget class="KPushButton" name="_addProperty">
        <property name="text">
         <string>add</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KColorCombo</class>
   <extends>QComboBox</extends>
   <header>kcolorcombo.h</header>
  </customwidget>
  <customwidget>
   <class>KDoubleNumInput</class>
   <extends>QWidget</extends>
   <header>knuminput.h</header>
  </customwidget>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
  <customwidget>
   <class>KPushButton</class>
   <extends>QPushButton</extends>
   <header>kpushbutton.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>_name</tabstop>
  <tabstop>_style</tabstop>
  <tabstop>_color</tabstop>
  <tabstop>_showName</tabstop>
  <tabstop>_showValue</tabstop>
  <tabstop>_propertiesTable</tabstop>
  <tabstop>_propertyName</tabstop>
  <tabstop>_propertyValue</tabstop>
  <tabstop>_isPropertyGlobal</tabstop>
  <tabstop>_addProperty</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>_propertyName</sender>
   <signal>returnPressed()</signal>
   <receiver>_addProperty</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>103</x>
     <y>275</y>
    </hint>
    <hint type="destinationlabel">
     <x>300</x>
     <y>277</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_propertyValue</sender>
   <signal>returnPressed()</signal>
   <receiver>_addProperty</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>214</x>
     <y>286</y>
    </hint>
    <hint type="destinationlabel">
     <x>289</x>
     <y>287</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
